frontend/src/metabase/components/LoadingAndErrorWrapper/LoadingAndErrorWrapper.jsx

```diff
@@ -134,7 +136,11 @@ export default class LoadingAndErrorWrapper extends Component {
       return Children.only(children);
     }
     return (
-      <div className={this.props.className} style={this.props.style}>
+      <div
+        className={this.props.className}
+        style={this.props.style}
```

Comment:
While we're at it, we can use destructuring for `className` and `style` like for all the other props.